Translate formalia/body.md (#102)
datasektionen/bawang-content: en/organization/formalia/body.md

Formalities
--------

> A collection of formal documents of the Computer Science Chapter.

Reviewer: meningsbyggnad
```suggestion
A collection of the Computer Science Chapter's formal documents.
```

Author: Ja, den var rätt så kass måste jag medge, tack!

> En samling av Datasektionens formella dokument.

Reviewer: Missade svensk rad
```suggestion

```

Author: Tack, missade den.

### [Protocols](/organisation/protokoll)

> Protocols from Chapter and Board Meetings

Reviewer: ```suggestion
Protocols from Chapter and Board Meetings (Swedish only)
```

Author: Bra ändring, tack!

### [Graphical profile](/organisation/grafisk-profil)

How can you actually use the logo of the Computer Science Chapter and 

> where can you find it in print-friendly formats?

Reviewer: How may the chapter logo be used and where can it be found in good formats?

### [Bylaws](https://styrdokument.datasektionen.se/stadgar)

The bylaws of the Computer Science Chapter, the latest revisions are 

> always available (in Swedish only, however).

Reviewer: ```suggestion
always available (in Swedish only).
```

Author: Ja, onödigt ord

### [Regulations](https://styrdokument.datasektionen.se/reglemente)

The Regulations regulate Chapter activities wherever the bylaws don't apply.

### [Equality Policy](https://styrdokument.datasektionen.se/jamlikhetspolicy)

The current equality policy of the Chapter.

### [Recruitment policy](https://styrdokument.datasektionen.se/rekryteringspolicy)

> Our recruitment policy explains and regulates our election and recruitment processes.

Reviewer: Förra lät som att den reglerar SM-val också.

```suggestion
Our recruitment policy explains and regulates our processes for recruitment to committees. (Swedish only)
```

Author: Rimligt, bättre ordval var detta.

Rekryteringspolicyn definierar och reglerar val, inval och rekrytering

> på sektionen.

Reviewer: Ta bort extra rader

Author: Yes please

### [Honorary Member](/sektionen/hedersmedlem)

A list of all persons whom have been appointed as honorary members of the Computer Science Chapter.

### [Operational plan](/organisation/verksamhetsplan)

The current operational plan of the Chapter.

### [Alcohol and drug policy](https://styrdokument.datasektionen.se/alkoholpolicy)

The current alcohol and drug policy of the Chapter.

### [DFunk-policy](https://styrdokument.datasektionen.se/dfunkpolicy)

The current dFunk-policy of the Chapter.

### [Economical regulatory documents](https://styrdokument.datasektionen.se/ekonomiskt_styrdokument)

The current economical regulatory documents of the Chapter.

### [Budget](/organisation/budget)

The current budget of the Chapter.

### [Annual reports](/organisation/arsrapporter)

> Operational plans and book keeping of past years (where there are).

Reviewer: "de år som har det"-biten fick försvinna. Det är även så att det finns för typ alla år i modern tid.
```suggestion
Annual reports and final accounts for past operational years.
```

Author: Ja, bättre ordval ger klarhet
